source/DSCResources/DSC_CMSiteConfiguration/DSC_CMSiteConfiguration.psm1

```diff
@@ -0,0 +1,1232 @@
+            'Number of Retries'        { $retry = $item.Value }
+            'Retry Delay'              { $retryDelay = $item.Value }
+        }
+    }
+
+    $siteDef = Get-CMSiteDefinition -SiteCode $SiteCode
+    $comments = ($siteDef.Props | Where-Object -FilterScript {$_.PropertyName -eq 'Comments'}).Value1
+    $defaultSize = ($siteDef.Props | Where-Object -FilterScript {$_.PropertyName -eq 'Device Collection Threshold'}).Value1
+    $maxSize = ($siteDef.Props | Where-Object -FilterScript {$_.PropertyName -eq 'Device Collection Threshold'}).Value2
+
+    if (($siteDef.Props | Where-Object -FilterScript {$_.PropertyName -eq 'Device Collection Threshold'}).Value -eq 0)
+    {
+        $siteServerDeployment = 'Block'
+    }
+    else
+    {
+        $siteServerDeployment = 'Warn'
+    }
+
+    # Communication Security
+    $comSec = Get-CMSiteComponent -ComponentName 'SMS_Site_Component_Manager' -SiteCode $SiteCode
+    $clientComms = ($comSec.Props | Where-Object -FilterScript {$_.PropertyName -eq 'IISSSLState'}).Value
+
+    if ($clientComms -eq 0)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $true
+        $pkiClient = $true
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 31)
+    {
+        $clientCommunication = 'HttpsOnly'
+        $useCrl = $false
+        $pkiClient = $true
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 63)
+    {
+        $clientCommunication = 'HttpsOnly'
+        $useCrl = $true
+        $pkiClient = $true
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 192)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $false
+        $pkiClient = $false
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 224)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $true
+        $pkiClient = $false
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 448)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $false
+        $pkiClient = $true
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 480)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $true
+        $pkiClient = $true
+        $sccmCert = $false
+    }
+    elseif ($clientComms -eq 1216)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $false
+        $pkiClient = $false
+        $sccmCert = $true
+    }
+    elseif ($clientComms -eq 1248)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $true
+        $pkiClient = $false
+        $sccmCert = $true
+    }
+    elseif ($clientComms -eq 1504)
+    {
+        $clientCommunication = 'HttpsOrHttp'
+        $useCrl = $true
+        $pkiClient = $true
+        $sccmCert = $true
+    }
+
+    foreach ($item in $comSec.Props)
+    {
+        switch ($item.PropertyName)
+        {
+            'Certificate Store'              { if ([string]::IsNullOrEmpty($item.Value1))
```

Comment:
The other instance of switch with if has the if on a new line, either move that to top line of move this to new line to match

Reply:
Fixed in local repo.